src/BizHawk.Client.Common/lua/CommonLibs/MemoryLuaLibrary.cs

```diff
@@ -47,12 +47,23 @@ public MemoryLuaLibrary(IPlatformLuaLibEnv luaLibsImpl, ApiContainer apiContaine
 		public void WriteByte(long addr, uint value, string domain = null) => APIs.Memory.WriteByte(addr, value, domain);
 
 		[LuaMethodExample("local nlmemrea = memory.readbyterange( 0x100, 30, mainmemory.getname( ) );")]
-		[LuaMethod("readbyterange", "Reads the address range that starts from address, and is length long. Returns the result into a table of key value pairs (where the address is the key).")]
+		[LuaMethod("readbyterange", "Reads the address range that starts from address, and is length long. Returns the result into a table of key value pairs (where the keys go from 0 to length-1).")]
 		public LuaTable ReadByteRange(long addr, int length, string domain = null) => _th.ListToTable(APIs.Memory.ReadByteRange(addr, length, domain));
 
+		[LuaMethodExample("")]
+		[LuaMethod("writebyterangeataddress", "Writes the given values as a contiguous block of unsigned bytes to a specific address")]
+		public void WriteByteRangeAtAddress(LuaTable memoryblock, long addr, string domain = null)
+		{
+			int i = 0;
+			foreach (var v in _th.EnumerateValues<double>(memoryblock))
+			{
+				APIs.Memory.WriteByte(addr+i, (uint)v, domain);
+				i++;
+			}
+		}

```

Comment:
This is out of scope, and we haven't reached a consensus re: #2844 yet.

Reply:
Ok, I am gonna remove this function!